packages/core/helper-plugin/lib/src/components/RelationInput/RelationInput.js

```diff
@@ -92,23 +94,22 @@ export const RelationInput = ({
                   <Box paddingTop={1} paddingBottom={1}>
                     {href ? (
                       <BaseLink disabled={disabled} href={href}>
-                        {title}
+                        {mainField}
                       </BaseLink>
                     ) : (
-                      title
+                      mainField
                     )}
                   </Box>
 
-                  {isDraft !== undefined && (
-                    <Badge
-                      borderSize={1}
-                      borderColor={`${badgeColor}200`}
-                      backgroundColor={`${badgeColor}100`}
-                      textColor={`${badgeColor}700`}
-                    >
-                      {isDraft ? 'Draft' : 'Published'}
-                    </Badge>
-                  )}
+                  {/* TO FIX: not showing badge if D&P is not activated  */}
+                  <Badge
+                    borderSize={1}
+                    borderColor={`${badgeColor}200`}
+                    backgroundColor={`${badgeColor}100`}
+                    textColor={`${badgeColor}700`}
+                  >
+                    {isDraft ? 'Draft' : 'Published'}
```

Comment:
We will need to translate these. Since we might move this component one day into the DS I think I'd need to pass you each of these labels. How would you prefer to receive them?

Since they don't need to be passed on each relation I think this should be a global prop for `RelationInput`?

Reply:
With the change in https://github.com/strapi/strapi/pull/14092/commits/add0c9f54f914204ef3a7d89b2a24402bd5ada0b I think we could solve this by passing another global prop to `RelationInput`:

```js
publicationStateTranslations={{
  draft: PropTypes.string.isRequired,
  published: PropTypes.string.isRequired
}}
```

Wdyt?